SalaryService.DataAccess/DependencyInjection.cs

```diff
@@ -2,20 +2,27 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
+
 namespace SalaryService.DataAccess;
 
 public static class DependencyInjection
 {
+    private const string DefaultConnection = "DefaultConnection";
+
     public static IServiceCollection AddPersistence(this IServiceCollection services, IConfiguration configuration)
     {
-        var connectionString = configuration.GetConnectionString("DefaultConnection");
+        var connectionString = configuration.GetConnectionString(DefaultConnection);
 
         services.AddDbContext<EmployeeDbContext>(options =>
         {
-            options.UseNpgsql(connectionString, o => o.UseNodaTime());
-        });
+            options.UseNpgsql(connectionString!,
```

Comment:
don't use `connectionString!`. If you want to check that the connection string is not null, do it above

Reply:
i'll do it later